src/testFixtures/java/uk/gov/di/ipv/cri/common/library/client/CommonApiClient.java

```diff
@@ -122,33 +94,12 @@ public HttpResponse<String> sendNewSessionRequest(String sessionRequestBody)
         return sendHttpRequest(request);
     }
 
-    public HttpResponse<String> sendTokenRequest(String privateKeyJwt)
-            throws IOException, InterruptedException {
-        var request =
-                HttpRequest.newBuilder()
-                        .uri(
-                                new URIBuilder(
-                                                this.clientConfigurationService
-                                                        .getPublicApiEndpoint())
-                                        .setPath(
-                                                this.clientConfigurationService.createUriPath(
-                                                        "token"))
-                                        .build())
-                        .header(HttpHeaders.CONTENT_TYPE, "application/x-www-form-urlencoded")
-                        .header(
-                                HttpHeaders.API_KEY,
-                                this.clientConfigurationService.getPublicApiKey())
-                        .POST(HttpRequest.BodyPublishers.ofString(privateKeyJwt))
-                        .build();
-        return sendHttpRequest(request);
-    }
-
-    public HttpResponse<String> sendNewTokenRequest(
-            PrivateKeyJWT privateKeyJwt, String code, String issuer)
+    public HttpResponse<String> sendTokenRequest(PrivateKeyJWT privateKeyJwt, String code)
             throws IOException, InterruptedException, URISyntaxException {
         var authorisationGrant =
                 new AuthorizationCodeGrant(
-                        new AuthorizationCode(code), new URI(issuer + "/callback"));
+                        new AuthorizationCode(code),
+                        new URI("ipv-core-stub-aws-headless/callback"));
```

Comment:
Would it be possible to make the issuer and redirect_uri parameters?

Reply:
I agree with @surajkumar, when we paired  and `new URI(issuer + "/callback")` used this, it was b'cos it was easier at the time as we didn't have the getParameter.. byName method. Try and refactor that method to take an argument so you can use it to get the `privatekeyKey`, `issuer` or `redirectUrl` value from parameter store